### PR TITLE
Adding option to require all translations

### PR DESCRIPTION
**Pull request by bakatrouble**

### CODE REVIEW

File: tests/test_forms.py [hunk 1]
```diff
@@ -46,6 +46,23 @@ def test_not_required():
     f.clean(['', ''])
 
 
+def test_all_required():
+    f = I18nFormField(widget=I18nTextInput, all_required=True)
+    assert f.clean(['A', 'B', 'C'])
+    with pytest.raises(ValidationError):
+        assert f.clean(['A', '', ''])
```
Comment (raphaelm) [inline tests/test_forms.py]: You need to have a separate ``with`` block for *each* of them, as the block will be left after the first exception is raised. Also, you can drop the ``assert`` keyword here.

File: tests/test_forms.py [hunk 2]
```diff
@@ -46,6 +46,23 @@ def test_not_required():
...
+        assert f.clean(['', 'B', ''])
+        assert f.clean(['', '', 'C'])
+
+
+def test_all_required_limited_locales():
+    f = I18nFormField(widget=I18nTextInput, all_required=True, locales=['de', 'fr'])
+    assert f.clean(['A', 'B'])
+    with pytest.raises(ValidationError):
+        assert f.clean(['A', ''])
+        assert f.clean(['', 'B'])
```
Comment (raphaelm) [inline tests/test_forms.py]: Same here

**Comment by bakatrouble** [comment]
Is it OK now?
I have never used `unittest` before

**Comment by bakatrouble** [comment]
Oops, doesn't work with locales passed to `I18nForm` constructor, fixing...

**Comment by bakatrouble** [comment]
Should work fine now

**Comment by raphaelm** [comment]
Looks good to me! Thank you very much!

**Comment by bakatrouble** [comment]
Could you please upload new version to PyPi so I could drop my repo?

**Comment by raphaelm** [comment]
Will try to do so soon!

**Comment by raphaelm** [comment]
Done

**Comment by bakatrouble** [comment]
Thanks!

**Pull request merged**